.gitattributes

```diff
@@ -0,0 +1,2 @@
+# Declare files that will always have CRLF line endings on checkout.
+* text eol=crlf
```

Comment:
Nu skriver jeg lige en kommentar fordi jeg kan. har vi gjort det rigtige i forhold til lineendings og denne: https://docs.github.com/en/free-pro-team@latest/github/using-git/configuring-git-to-handle-line-endings.